helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/templates/daemonset.yaml

```diff
@@ -79,10 +80,7 @@ spec:
               fieldRef:
                 fieldPath: metadata.name
           - name: SPLUNK_HEC_TOKEN
-            valueFrom:
-              secretKeyRef:
-                name: {{ template "splunk-kubernetes-logging.secret" . }}
-                key: splunk_hec_token
```

Comment:
this may not be necessary.
would it work if we:
 - keep the `splunk-kubernetes-logging.secret` object and `splunk_hec_token` key
 - set its value to be `od_secret:<vault_path_to_hec_token>`

`od-secret` is capable of interpreting that early enough and this set up should work.

Reply:
Good question. I'll try it in a bit. It will set the env var to the od-secrer

Reply:
btw, so the reason why I did this is so that we can keep the token in 1 place, as opposed to definining it for each chart.